### PR TITLE
add more targets, sort list alphabetically

### PR DESCRIPTION
**Pull request by silverwind**
Added a few more target files, notably `ChangeLog` which was encountered.

@Trott

### CODE REVIEW

File: lib/targets.js [hunk 1]
```diff
@@ -53,14 +52,23 @@ module.exports = [
...
+    'Gruntfile.*',
     'gruntfile.*',
+    'Gulpfile.*',
     'gulpfile.*',
-    'npm-debug.log',
+    'HISTORY',
+    'History',
+    'HISTORY.*',
+    'History.*',
+    'npm-debug.log*',
```
Comment (silverwind) [inline lib/targets.js]: This file is known to also have suffixes

**Comment by BridgeAR** [comment]
Maybe include README.md as well (and it's forms)?

**Comment by silverwind** [comment]
> Maybe include README.md as well (and it's forms)?

There's the notion that `README.md` may contain licencing information, so not sure if this is a good idea. It's also one of those files that npm will never exclude in a publish.

**Comment by Trott** [comment]
Released in dmn@2.0.0.

**Pull request merged**